pilot/pkg/xds/statusgen.go

```diff
@@ -106,6 +107,11 @@ func isProxy(con *Connection) bool {
 		con.proxy.Metadata.ProxyConfig != nil
 }
 
+func isZtunnel(con *Connection) bool {
+	return con != nil && con.proxy != nil &&
+		con.proxy.Metadata != nil && strings.HasPrefix(con.proxy.ID, "ztunnel")
```

Comment:
I would check con.proxy.Type

Reply:
Thanks @hanxiaop , it has been changed. :-)